applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -200,15 +209,15 @@ class MapperVertexMorphing
             NodeVector neighbor_nodes( mMaxNumberOfNeighbors );
             std::vector<double> resulting_squared_distances( mMaxNumberOfNeighbors );
             unsigned int number_of_neighbors = mpSearchTree->SearchInRadius( node_i,
-                                                                             mFilterRadius, 
+                                                                             mFilterRadius,
                                                                              neighbor_nodes.begin(),
-                                                                             resulting_squared_distances.begin(), 
+                                                                             resulting_squared_distances.begin(),
                                                                              mMaxNumberOfNeighbors );
 
             std::vector<double> list_of_weights( number_of_neighbors, 0.0 );
             double sum_of_weights = 0.0;
 
-            ThrowWarningIfMaxNodeNeighborsReached( node_i, number_of_neighbors );                                                                               
+            ThrowWarningIfMaxNodeNeighborsReached( node_i, number_of_neighbors );
             ComputeWeightForAllNeighbors( node_i, neighbor_nodes, number_of_neighbors, list_of_weights, sum_of_weights );
```

Comment:
Thanks, Pooyan for that hint. About the KDTree, there is no special reason for using it. We just know how to handle it, unlike the bin-search, which we never used before. Since I am not an expert in spatial search, can you recommend using the bin search for this case, like do you expect performance improvements? 

Any advice is appreciated!